fixtures/cassettes/hand-edited_requests.yml

```diff
@@ -295,6 +295,19 @@ http_interactions:
   digest: invalid_but_load_page
 - request:
     method: get
+    uri: https://cmr.earthdata.nasa.gov/search/collections.json?include_facets=true&page_size=20&page_num=1&include_has_granules=true&include_granule_counts=true
```

Comment:
I saw in the test log that it tries to find this fixture in hand_edited in after :all. So I put it there and the error is gone. Well, for most of the time, VCR just complains but the tests won't fail.